config/routes.rb

```diff
@@ -327,6 +327,12 @@
       post '/forgot_password' => 'forgot_password#update'
       get '/document_capture' => 'document_capture#show'
       put '/document_capture' => 'document_capture#update'
+      # This route is included in SMS messages sent to users who start the IdV hybrid flow. It
+      # should be kept short, and should not include underscores ("_").
+      get '/documents', to: redirect('/verify/hybrid_mobile'),
+                        as: :hybrid_mobile_document_capture_entry
```

Comment:
Seems like this might get confusing since we have a separate document_capture controller.
```suggestion
                        as: :hybrid_mobile_entry
```